Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -154,30 +154,36 @@ - (void)tearDown {
 #pragma mark - Method Swizzling Tests
 
 - (void)testSwizzlingNonAppDelegate {
-  id randomObject = @"Random Object that is not an App Delegate";
-  [self.proxy swizzleAppDelegateMethods:randomObject];
-  XCTAssertFalse(self.proxy.didSwizzleAppDelegateMethods);
-}
+  RandomObject *invalidAppDelegate = [[RandomObject alloc] init];
```

Comment:
Your solution for each tests work but I'm not sure what's wrong with the original one?
We need a good reason of refactoring existing tests, not just for the sake of refactor. It takes time for writing, reviewing and maintaining the release for big changes.

Reply:
Talked offline and looks like we only want to use public API for testing to be able to switch the new swizzling class. Will close some of the conversations.